service/Cargo.toml

```diff
@@ -62,7 +62,9 @@ tokio = { version = "1.22.0", features = ["macros"] }
 uuid = { workspace = true, features = ["v4"] }
 
 [features]
-codegen = ["shuttle-codegen"]
+default = ["codegen"]
```

Comment:
I removed this from default in my merge of main, which I think was a mistake since all dependents of shuttle-service should need it. Thanks! :sweat_smile: